changelog.md

```diff
@@ -17,6 +17,8 @@
 - [#3724](https://github.com/ignite/cli/pull/3724) Add or vendor proto packages from Go dependencies
 - [#3715](https://github.com/ignite/cli/pull/3715) Add test suite for the cli tests
 - [#3756](https://github.com/ignite/cli/pull/3756) Add faucet compatibility for latest sdk chains
+- [#3802](https://github.com/ignite/cli/pull/3802) Remove error from analytics
```

Comment:
I don't think it needs a changelog, as analytics never got released

Reply:
removed